app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  validates :title, length: { maximum: 100 }, presence: true
+  validates :text, length: { maximum: 300 }, presence: true
+
+  after_save :update_post_counter
+
+  def return_last_five_comments
+    comments.last(5)
+  end
+
+  private
+
+  def update_post_counter
+    author.update(posts_count: author.posts.count)
+  end
```

Comment:
- [OPTIONAL] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to use the `increment` method to increment the counter every time a new post has been saved to database👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :


   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```

Reply:
It would be faster, but it could cause the counter to be off-sync with the actual count of posts

Reply:
[OPTIONAL] It would be best if you adopted a `growth mindset` and showed a willingness to adopt new solutions if they are more effective and improve the performance of your application. 😊